### ISSUE TITLE
relatve dates

### ISSUE THREAD

**Issue by cjohnsonuk** [maphy-psd/dokuwiki-plugin-date#4]
Not sure how this is working (I'm running on MAMP with php7

{{date>%Y-%m-%d %H:%M|timestamp=strtotime("now")}}

{{date>%Y-%m-%d %H:%M|timestamp=strtotime("now +1 day")}}

results in 
2016-04-29 15:27

2016-05-01 15:27

instead of 

2016-04-29 15:27

2016-04-30 15:27

trying to test on an earlier version of php but that box is offline atm

**Comment by maphy-psd** [comment]
This is really strange. With php54 same behavior, but only with the plugin. I don't figure out the issue. I think this isn't a bug in php. 

As a workaround you can use the `now` parameter

```
{{date>%Y-%m-%d %H:%M|timestamp=strtotime("now")|now=strtotime("next day")}}
```

**Comment by maphy-psd** [comment]
fixed by 1d67b8d
